Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptEditController.java

```diff
@@ -258,6 +272,43 @@ public ModelAndView confirmEdit(@PathVariable("id") String id, HttpServletReques
         return new ResponseEntity<String>(buildJSON(synonymList, false, true), HttpStatus.OK);
     }
 
+    @RequestMapping(method = RequestMethod.GET, value = "conceptEdit/search")
+    public @ResponseBody ResponseEntity<Object> searchConcept(@RequestParam("concept") String concept,
```

Comment:
Hm, ya, two lines are the same and the structure... I'll create a new ticket to refactor the wrapper creator. For now let's leave it the way it is.